app/views/incomeSources/add/IncomeSourceAddedObligations.scala.html

```diff
@@ -0,0 +1,220 @@
+@getMessage(key: String, args: String*) = @{
+  messages(s"business-added.$key", args: _*)
+}
+
+
+@getPageTitle = @{
+
+    incomeSourceType match {
+        case _:SelfEmployment.type =>
+            s"""${getMessage("sole-trader.head")} ${businessName.get} ${getMessage("sole-trader.base")}""".trim()
+
+        case _:UkProperty.type =>
+            s"${getMessage("uk-property.head")} " +
+            s"${getMessage("uk-property.h1")} " +
+            s"${getMessage("uk-property.base")}".trim()
+
+        case _:ForeignProperty.type =>
+            s"""${getMessage("foreign-property.h1")} ${getMessage("foreign-property.base")}"""
+    }
+}
+
+@getAddIncomeSourcesCall = @{
+    if(isAgent) {
+        controllers.incomeSources.add.routes.AddIncomeSourceController.showAgent()
+    } else {
+        controllers.incomeSources.add.routes.AddIncomeSourceController.show()
+    }
+}
+
+@bannerHtmlContent(head:String, h1:String, base:String) = {
+    <div class="govuk-panel__body govuk-!-margin-bottom-3">
+        @head
+    </div>
+    <h1 class="govuk-panel__title">
+        @h1
+    </h1>
+    <div class="govuk-panel__body">
+        @base
+    </div>
+}
+
+@getBannerContent() = @{
+    incomeSourceType match {
+        case _:SelfEmployment.type =>
+            bannerHtmlContent(getMessage("sole-trader.head"),businessName.get,getMessage("sole-trader.base"))
+        case _:UkProperty.type =>
+            bannerHtmlContent(getMessage("uk-property.head"),getMessage("uk-property.h1"),getMessage("uk-property.base"))
+        case _:ForeignProperty.type =>
+        bannerHtmlContent(getMessage("foreign-property.head"),getMessage("foreign-property.h1"),getMessage("foreign-property.base"))
+    }
+}
+
+
+
+@import implicitDateFormatter.longDate
+@mainTemplate(pageTitle = getPageTitle,
+  backUrl = Some(backUrl),
+  isAgent = isAgent,
+  btaNavPartial = user.btaNavPartial) {
+
+    @formWithCSRF(action = getAddIncomeSourcesCall) {
+
+        <div class="govuk-panel govuk-panel--confirmation" id="banner">
+            @getBannerContent()
+        </div>
+
+        <h2 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--l govuk-!-margin-top-8">
+                @getMessage("text")
+            </label>
+        </h2>
+
+
+        @if(sources.quarterlyObligationsDatesYearOne.nonEmpty){
+        <div class="box-simple" id="quarterly">
+            <div class="page_headers govuk-heading-m"> @getMessage("quarterly-heading") </div>
+            <p class="govuk-body"> @getMessage("quarterly-p1") </p>
+
+            <table class="govuk-table">
+                <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header govuk">@(getMessage("table.heading-1")+" "+sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear+" "+getMessage("to")+" "+(sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear + 1))</th>
```

Comment:
You could improve readability by adding spaces after the +s

Reply:
Actually this causes a warning: method + in class Int is deprecated (since 2.13.0): Adding a number and a String is deprecated. Use the string interpolation `s"$num$str"`